metadata filtering support (#21870)
cloudflare/cloudflare-docs: src/content/docs/autorag/how-to/simple-search-engine.mdx

---
pcx_content_type: concept
title: Create a simple search engine
sidebar:
  order: 5
---

> Using the `search` method you can implement a simple but fast search engine. This example is done using [Workers Binding](/autorag/usage/workers-binding/) but can be easily adapted to use the [REST API](/autorag/usage/rest-api/) instead.

Reviewer: ```suggestion
By using the `search` method, you can implement a simple but fast search engine. This example uses [Workers Binding](/autorag/usage/workers-binding/), but can be easily adapted to use the [REST API](/autorag/usage/rest-api/) instead.
```

To replicate this example remember to:

> - Disable `rewrite_query` as you want to match the original user query

Reviewer: ```suggestion
- Disable `rewrite_query`, as you want to match the original user query
```
